src/Core/Components/Tabs/FluentTab.razor.cs

```diff
@@ -148,7 +152,7 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
     /// <summary />
     protected virtual Task CloseClickedAsync()
     {
-        return Owner!.UnregisterTabAsync(this);
+        return Owner!.UnregisterTabAsync(Id!);
```

Comment:
Same as my other comment : `Id` is nullable.

Reply:
Same comment for me as well 😁